docs/src/main/asciidoc/datasource.adoc

```diff
@@ -85,7 +85,17 @@ quarkus.datasource.jdbc.max-size=16
 <1> This configuration value is only required if there is more than one database extension on the classpath.
 
 If only one viable extension is available, Quarkus assumes this is the correct one.
-If a driver has been added to the test scope, Quarkus automatically includes the driver in testing.
+When a driver has been added to the test scope, Quarkus automatically includes the driver in testing.
+
+==== JDBC connection pool size adjustment
+
+To protect your database from overloading during the transaction load peaks, size the pool adequately to throttle the database load.
+The proper size always depends on many factors, such as the number of parallel application users or the nature of the workload.
+
+Be aware that setting the pool size too low might cause some requests to time out while waiting for a connection.
+
+For more information about pool size adjustment properties, see the xref:jdbc-configuration[] section.
```

Comment:
For ease of doc maintenance and automatic title updates in links, use `<<section-ID>>` instead of xrefs.

```suggestion
For more information about pool size adjustment properties, see the <<jdbc-configuration>> section.
```

_The downstreamdoc.java postprocessor build job converts these to xrefs for PV2 consumption._